stylelint-config.json

```diff
@@ -13,6 +14,9 @@
       "message": "The & operator is not allowed at the end of theme selectors.",
       "filePattern": "-theme\\.scss$"
     }],
+    "material/no-concrete-rules": [true, {
+      "filePattern": "^_.*\\.scss$|-theme\\.scss$"
```

Comment:
Why not just `^_.*\\.scss$`?

Reply:
I wanted to capture cases like `something-theme.scss`, but I guess it's redundant since all the theme files are prefixed.